types/jsdom/index.d.ts

```diff
@@ -6,9 +6,10 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference path="ts3.5/index.d.ts"/>
+/// <reference path="./base.d.ts"/>
```

Comment:
Why does the main index need to reference both base and ts3.5's index? I thought the point of the change was to keep the common core in base.